articles/backup/backup-azure-vms-automation.md

```diff
@@ -523,6 +523,47 @@ A user can selectively restore few disks instead of the entire backed up set. Pr
 
 Once you restore the disks, go to the next section to create the VM.
 
+#### Restore disks to Secondary region
+
+If CRR is enabled on the vault with which you have protected your VMs, the backup data is replicated to the secondary region and hence can be used to perform a restore. Perform the below steps to trigger restore in secondary region:
+
+* Step 1: [Fetch the vault with which you have protected your VMs](backup-azure-vms-automation.md#fetch-the-vault-id)
+* Step 2: [Select the right backup item  for which you want to trigger the restore](backup-azure-vms-automation#select-the-vm-when-restoring-files)
+* Step 3: Select the appropriate recovery point in secondary region that you want to use for performing restore
+
+```powershell
+$rp=Get-AzRecoveryServicesBackupRecoveryPoint -UseSecondaryRegion -Item $backupitem -VaultId $targetVault.ID
+$rp=$rp[0]
+```
+
+* Step 4: Execute [Restore-AzRecoveryServicesBackupItem](/powershell/module/az.recoveryservices/restore-azrecoveryservicesbackupitem) cmdlet with -RestoreToSecondaryRegion parameter to trigger restore in secondary region
```

Comment:
* Step 4: Execute the [Restore-AzRecoveryServicesBackupItem](/powershell/module/az.recoveryservices/restore-azrecoveryservicesbackupitem) cmdlet with the **-RestoreToSecondaryRegion** parameter to trigger a restore in the secondary region